.editorconfig

```diff
@@ -30,6 +30,11 @@ insert_final_newline = false
 indent_style = space
 indent_size = 2
 
+[*.sql]
+insert_final_newline = true
```

Comment:
@mukeshpanchal27 don't believe this is needed as it is inherited by the `[*]` rules.

Reply:
Thanks for pointing it out.